contrib/features/TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+# Change Request
```

Comment:
when we do templates like this in markdown, i prefer instructions to be in comment form.  i wonder what you all think?